mocks/DUMMY_DATA.tsx

```diff
@@ -172,7 +172,6 @@ export const navigation = {
   pages: [
```

Comment:
are we still using it or you just left it here in case someone will want to use it?

Reply:
Just in case for testing we will eventually delete it.